packages/atlas/src/components/YppReferralTable/YppReferralTable.tsx

```diff
@@ -104,15 +105,19 @@ const Tier = ({ yppStatus }: { yppStatus: YppChannelStatus }) => {
   )
 }
 
-const Reward = ({ yppStatus }: { yppStatus: YppChannelStatus }) => {
+const Reward = ({ yppStatus, signupTimestamp }: { yppStatus: YppChannelStatus; signupTimestamp: number }) => {
+  const multiplier =
+    yppBackendTierToConfig(yppStatus) !== 'bronze' && signupTimestamp > BOOST_TIMESTAMP
+      ? atlasConfig.features.ypp.tierBoostMultiplier || 1
+      : 1
   return (
     <RightAlignText as="p" variant="t100-strong">
       {yppStatus.startsWith('Suspended')
         ? 'Not paid'
         : yppStatus === 'Unverified'
         ? 'Pending'
         : yppStatus.startsWith('Verified')
-        ? `$${getTierRewards(yppBackendTierToConfig(yppStatus))?.referral}`
+        ? `$${(getTierRewards(yppBackendTierToConfig(yppStatus))?.referral || 0) * multiplier}`
```

Comment:
Maybe I'm just confused about what this "reward" means but just to make sure: won't this make the silver reward `10 * 2 = 20` instead of `25` as mentioned in  #5187 ?